### ISSUE TITLE
Evaluation on 1/3 test dataset

### ISSUE THREAD

**Issue by DiligentPanda** [ruotianluo/self-critical.pytorch#117]
Hi, thanks for your elegant code. I notice that in https://github.com/ruotianluo/self-critical.pytorch/blob/master/eval_utils.py there is a line `# filter results to only those in MSCOCO validation set (will be about a third)`. This is a potential problem, because papers use your codes might report this performance on a subset of the test dataset.

**Comment by ruotianluo** [comment]
Thanks for your catch. The scores are correct. The comment is copied from neuraltalk2 which is different and is not correct in my code. I will delete what's in the parenthesis.
